azure-pipelines.yml

```diff
@@ -85,8 +85,9 @@ stages:
             swig libmpc-dev
           displayName: 'Install dependencies'
         - script: |
-            python -m pip install --upgrade build pip wheel
-            python -m pip install asteval==0.9.22 numpy==1.18 scipy==1.4.0 uncertainties==3.1.4
+            python -m pip install --upgrade pip wheel
+            pip install setuptools==57.5.0
```

Comment:
arggghhh.... seuptools is breaking things again :( It looks like NumPy pins their version to 59.2.0, but even if I try that I still see failures in the CI - similar to here in this PR. 

I'll try and fix that, so don't worry about / waste time with that part - hopefully soon and then I'll rebase/push to this PR and it should be good to go...